staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/patch_test.go

```diff
@@ -428,6 +428,44 @@ testCases:
     modified:
       retainKeysMap:
         name: foo
+  - description: retainKeys map can delete an unexisting merging list
+    original:
+      retainKeysMap: {}
+    twoWay:
+      retainKeysMap:
+        mergingList:
+          - name: noxu 
+            $patch: delete
+    modified:
+      retainKeysMap: {}
+  - description: retainKeys map can replace an unexisting merging list
```

Comment:
This seems to be an breaking behavior change for `replace` directive.
IMHO there is no bug for `replace`. And this behavior change for `replace` seems wrong to me.
#67546 is for `delete` directive.